### DISCUSSION TITLE
PostgreSQL running on a different port

### DISCUSSION THREAD

**Discussion by ekhalifa-dev** [TomBursch/kitchenowl#415] [Q&A]
Hi,
I have my database running on a different system. and the port differs from the default 5432. Now, on the docker compose if I fill xxx.xxx.xxx.xxx:port as a DB_HOST value, I get  a "(psycopg2.OperationalError) could not translate host name" error. 
Any help would be appreciated.

**Comment by TomBursch** [comment]
Okay, this is a bit my fault, right now, there is no way to change the database port. I'll add it as soon as possible.

**Reply by ekhalifa-dev** [reply]
thanx

**Reply by TomBursch** [reply]
I'll close this discussion as you can now use `DB_PORT` for setting the port
